README.md

```diff
@@ -1,144 +1,103 @@
-
-```bash
-git clone https://github.com/fort-nix/nix-bitcoin
-cd nix-bitcoin/examples/
-nix-shell
-```
-
-The following example scripts set up a nix-bitcoin node according to [`examples/configuration.nix`](examples/configuration.nix) and then
-shut down immediately. They leave no traces (outside of `/nix/store`) on the host system.
-
-- [`./deploy-container.sh`](examples/deploy-container.sh) creates a [NixOS container](https://github.com/erikarvstedt/extra-container).\
-  This is the fastest way to set up a node.\
-  Requires: [Nix](https://nixos.org/), a systemd-based Linux distro and root privileges
-
-- [`./deploy-qemu-vm.sh`](examples/deploy-qemu-vm.sh) creates a QEMU VM.\
-  Requires: [Nix](https://nixos.org/nix/)
-
-- [`./deploy-nixops.sh`](examples/deploy-nixops.sh) creates a VirtualBox VM via [NixOps](https://github.com/NixOS/nixops).\
-  NixOps can be used to deploy to various other backends like cloud providers.\
-  Requires: [Nix](https://nixos.org/nix/), [VirtualBox](https://www.virtualbox.org)
-
-- [`./deploy-container-minimal.sh`](examples/deploy-container-minimal.sh) creates a
-  container defined by [minimal-configuration.nix](examples/minimal-configuration.nix) that
-  doesn't use the [secure-node.nix](modules/presets/secure-node.nix) preset.
-  Also shows how to use nix-bitcoin in an existing NixOS config.\
-  Requires: [Nix](https://nixos.org/), a systemd-based Linux distro and root privileges
-
-Run the examples with option `--interactive` or `-i` to start a shell for interacting with
-the node:
-```bash
-./deploy-qemu-vm.sh -i
-```
-
-#### Tests
-The internal test suite is also useful for exploring features.\
-The following `run-tests.sh` commands leave no traces (outside of `/nix/store`) on
-the host system.
+A Bitcoin node verifies the Bitcoin protocol and provides ways of interacting with the Bitcoin network. nix-bitcoin
+nodes are used for a variety of purposes and can serve as personal or merchant wallets, second layer public
+infrastructure and as backends for Bitcoin applications. In all cases, the aim is to provide security and privacy by
+default. However, while nix-bitcoin is used in production today, it is still considered experimental.
 
-```bash
-git clone https://github.com/fort-nix/nix-bitcoin
-cd nix-bitcoin/test
+A full installation of nix-bitcoin is usually deployed either on a dedicated (virtual) machine or runs in a container
+and is online 24/7. Alternatively, the Nix packages, NixOS modules and configurations can be used independently and
+combined freely.
 
-# Run a Python test shell inside a VM node
-./run-tests.sh debug
-print(succeed("systemctl status bitcoind"))
+nix-bitcoin is built on top of Nix and NixOS because that allows using abstractions powerful enough to keep it highly
+customizable and maintainable. Testament to this are nix-bitcoin's robust security features and its potent test
+framework.  However, running nix-bitcoin does not require any previous experience with the Nix ecosystem.
```

Comment:
Nit, make wording more concise and poignant:
```suggestion
nix-bitcoin is built on top of Nix and NixOS which provide powerful abstractions to keep it highly
customizable and maintainable. Testament to this are nix-bitcoin's robust security features and its potent test
framework.  However, running nix-bitcoin does not require any previous experience with the Nix ecosystem.
```

Reply:
Fixed in bcedf69549652b9d201cc9e02f64c086d6501d14